actions/event-actions/lib/common.js

```diff
@@ -132,7 +130,7 @@ function constructObject(data) {
                 jsonObject = JSON.parse(data);
             }
             catch (e) {
-                console.log('error parsing ' + data);
+                console.error('error parsing ' + data);
```

Comment:
What about this line? Would data ever have sensitive data?

Reply:
It's a good point, I'm not 100% sure either way. The only place where this is used is building `additionalData` when creating a trigger in the `changesWebAction`. My understanding is that that's a configuration object and wouldn't sensibly contain user data, but I'd be interested in others' analysis?